.github/workflows/tests.yml

```diff
@@ -39,6 +39,11 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Check types with mypy
+      run: |
+        python3 -m pip install mypy
```

Comment:
Ideally these should be in test dependencies and be installable in one line

Reply:
You mean inside `project.optional-dependencies` in pyproject.toml?

Reply:
I see - done this now.